example/vm/config.yaml

```diff
@@ -0,0 +1,201 @@
+                        - debugfs
+                        - devpts
+                        - devtmpfs
+                        - fusectl
+                        - hugetlbfs
+                        - iso9660
+                        - mqueue
+                        - nsfs
+                        - overlay
+                        - proc
+                        - procfs
+                        - pstore
+                        - rpc_pipefs
+                        - securityfs
+                        - selinuxfs
+                        - squashfs
+                        - sysfs
+                        - tracefs
+            memory:
+                metrics:
+                    system.memory.utilization:
+                        enabled: true
+            # paging:
+            # processes:
+            # process:
+            network: {}
+    otlp:
+    # OTLP is the default protocol supported by all language implementations.
+    # For more details, see
+    # https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/README.md
+        protocols:
+            grpc:
+                endpoint:
+            http:
+                endpoint:
+processors:
+    # We recommend use of the batch processor.  We recommend the settings
+    # below for traces.
+    #
+    # Note: We are aware of ongoing efforts within OpenTelemetry to
+    # configure batching in the exporter, where it is possible to
+    # configure batch size limits in terms of bytes, instead of items.
+    # We will update these recommendations when batching by size is
+    # available.
+    batch:
+        # In this example, the processor will wait to accumulate at least
+        # 1000 spans for up to 1 second, then flush the batch.  In cases
+        # where the arriving data is already batched, such that combining
+        # the pending batch with the arriving data would exceed 1500
+        # items, then 1500 items will be sent by splitting the data.
+        #
+        # Note: the batch processor has a side-effect of returning success
+        # to the producer, before waiting for the consumer to respond.
+        # This is appropriate default in most cases, it means that SDKs
+        # sending to the gateway will not see or report errors.
+        #
+        # The batch processor responds to "back-pressure" from the
+        # exporter, meaning it is never directly responsible for dropping
+        # spans.  Note that our current recommendation for exporter
+        # settings does not respond with back-pressure to the batch
+        # processor.  Due to exporter settings, this collector
+        # configuration will drop data when the ServiceNow service is
+        # (intentionally or accidentally) refusing data, instead of
+        # applying pressure backward, discussed in the `exporters`
+        # section.
+        send_batch_size: 1000
+        timeout: 1s
+        send_batch_max_size: 1500
+
+exporters:
```

Comment:
Description for exporters:
> An exporter is how data gets sent to different systems/back-ends. Generally, an exporter translates the internal format into another defined format.

Cloud Observability supports OTLP natively for all signals. Therefore the only required exporter is the OTLP exporter, which requires a Cloud Observability access token. The example extracts the access token from an environment variable, to avoid hard-coding this in the configuration file.